### PR TITLE
Houses count fix

### PR DESCRIPTION
**Pull request by Tolokio**
Fix for "remove" and "nuke" not working well with multis.

No script's fix is required now. Tested and working well with houses and ships.

EDIT:

DONE another PR. with better fix. I leave this open until @canerksk answers!!!

### CODE REVIEW

**Comment by canerksk** [comment]
Add a coowner to the house and then delete that added coowner character with remove 1, then after removing that house and taking backup, it crashed.

**Comment by AtillaG1** [comment]
could not reproduce it either.

File: src/game/CObjBase.cpp [hunk 1]
```diff
@@ -2395,6 +2395,21 @@ bool CObjBase::r_Verb( CScript & s, CTextConsole * pSrc ) // Execute command fro
 			return pClientSrc->addGumpDialogProps( GetUID() );
 		case OV_REMOVE:	//remove this object now.
 			EXC_SET_BLOCK("REMOVE");
+			if (IsItem())
+			{
+				CItem* pItem = static_cast <CItem*>(this);
+				if (pItem->IsTypeMulti())
```
Comment (cbnolok) [inline src/game/CObjBase.cpp:2401]: Couldn't this be done in citem::delete?

File: src/game/clients/CClientTarg.cpp [hunk 2]
```diff
@@ -652,6 +652,17 @@ bool CClient::OnTarg_Tile( CObjBase * pObj, const CPointMap & pt )
 
 				if ( m_Targ_Text.IsEmpty())
 				{
+					if (pItem->IsTypeMulti())
+					{
+						CItemMulti* pMulti = static_cast<CItemMulti*>(pItem);
```
Comment (cbnolok) [inline src/game/clients/CClientTarg.cpp:657]: This could be redundant if the check was moved into citem::delete

Comment (Tolokio) [reply src/game/clients/CClientTarg.cpp:657]: yes, can be done at to delete(), I prefer it as well.

**Pull request closed without merging**